src/CodeStyle/CSharp/Tests/Microsoft.CodeAnalysis.CSharp.CodeStyle.UnitTests.csproj

```diff
@@ -15,6 +15,10 @@
   </ItemGroup>
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeFix.Testing.XUnit" Version="$(MicrosoftCodeAnalysisCSharpCodeFixTestingXUnitVersion)" />
+
+    <!-- TODO: Remove NuGet version requirements once testing library moves to >= 6.7.0 -->
```

Comment:
Lets have a tracking issue to get that resolved.  What is currently blocking it?